middleware/http/wasm/internal/e2e-guests/config/main.go

```diff
@@ -0,0 +1,12 @@
+// Package main ensures tests can prove logging or stdio isn't missed, both
```

Comment:
We need to add the copyright header to all files please. I just notice it's missing in `httpwasm.go` and other files. This is the header:

```go
/*
Copyright 2023 The Dapr Authors
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at
    http://www.apache.org/licenses/LICENSE-2.0
Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/
```

Reply:
Done, added to all .go files.

This config is a host abi function provided by http-wasm, also the application can pass configs via request data and metadata to wasm binding, I think it is ok for not to add config for binding unless there are users ask to.